app/forms/provider_interface/application_data_export_form.rb

```diff
@@ -41,6 +41,12 @@ def at_least_one_recruitment_cycle_year_is_selected
       end
     end
 
+    def all_or_specific_status_selected
+      if application_status_choice.blank?
+        errors.add(:application_status_choice, 'Select a status type')
```

Comment:
for these forms, is there an official and approved way of pulling in this text from `en.yml`?

Reply:
I think it can be done like this https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/config/locales/provider_interface/reasons_for_rejection.yml#L76-L79